docs/en/structured-data/catalog-service.md

```diff
@@ -87,15 +90,15 @@ This command maps the hive database `default` into a database in Alluxio called
 the metastore located at `thrift://metastore_host:9083`
 
 ```console
-$ ${ALLUXIO_HOME}/bin/alluxio table attachdb alluxio_db hive \
-    --option hive.database-name=default \
-    --option hive.metastore.uris=thrift://metastore_host:9083
+$ ${ALLUXIO_HOME}/bin/alluxio table attachdb --db alluxio_db hive \
+    thrift://metastore_host:9083 default
 ```
 
 > **Note:** When databases are attached, all tables will be synced from the configured UDB.
 If out-of-band updates occur to the database or table and the user wants query results to reflect
 the updates, the user must detach ([See Detaching Databases](#detaching-databases)) and then
-re-attach the database to reflect up any updates.
+re-attach the database to reflect up any updates. Alternatively, if the only changes to the database
```

Comment:
new sentence should be on a newline

https://docs.alluxio.io/os/user/stable/en/contributor/Documentation-Conventions.html#line-breaks

Reply:
done